core/arch/arm/plat-stm32mp1/conf.mk

```diff
@@ -254,6 +254,15 @@ CFG_WITH_NSEC_UARTS ?= y
 # UART instance used for early console (0 disables early console)
 CFG_STM32_EARLY_CONSOLE_UART ?= 4
 
+# Disable the HUK by default as it requires CFG_STM32MP1_HUK_BSEC_BASE
+# which is product specific
+CFG_STM32MP1_HUK_BSEC ?= n
+ifeq ($(CFG_STM32MP1_HUK_BSEC),y)
+# An AES-GCM encryption key is retrieved from OTP and used to encrypt the UID
+# OTP in order to generate the HUK. Otherwise the HUK is read from OTP (default)
+CFG_STM32MP1_HUK_BSEC_DERIVE_GCM ?= n
```

Comment:
Nitpicking again: i would rather name it `CFG_STM32MP1_HUK_BSEC_DERIVE_DEVUID` as the purpose is more explicit, that is deriving HUK using device UID.

Reply:
sure I can do that. CFG_STM32MP1_HUK_BSEC_DERIVE_DEV_UID